### ISSUE TITLE
Interactive element on File Set does not meet minimum size nor spacing

### ISSUE THREAD

**Issue by posixeleni** [samvera/hyku#3060]
vpat.hykudemo.org
Version 7.1

### Descriptive summary

Link text "Download image" does not meet minimum size or spacing on File Set pages.

Interactive elements, like buttons or links, should be large enough or be positioned, so they can easily be clicked or tapped by users.

According to WCAG 2.2: [Opens in new tab:Understanding Success Criterion 2.5.8: Target Size Minimum](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum): Elements should be at least 24 by 24 pixels, or if smaller, be positioned with enough spacing to not intersect with other interactive elements.

Current size of element:119.18 x 17.60 pixels

### Rationale

Accessibility issue remediation

### Expected behavior

Enlarge the interactive element to be at least 24 by 24 pixels.

or

Adjust the spacing so a 24-pixel diameter circle centered on the bounding box of an undersized interactive element does not intersect with another interactive element or its circle.

### Actual behavior

Current size of element: 119.18 x 17.60 pixels which does not meet the at least 24 by 24 pixels requirement.

### Steps to reproduce the behavior

Go to this page: https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5
Load SiteImprove Accessibility Checker and see the size of the "Download image" link does not meet the minimum size requirement

<img width="1052" height="1057" alt="Image" src="https://github.com/user-attachments/assets/c6569d96-a7a8-4409-8698-e562baf2252d" />

**Issue can be found on these pages:**

https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=de
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=en
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=es
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=fr
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=it
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=pt-BR
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=zh
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=de
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=en
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=es
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=fr
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=it
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=pt-BR
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=zh

### Related work

Another accessibility issue on File Set pages about Links missing text alternative: https://github.com/samvera/hyku/issues/3047
